api/tests/unit/application/preHandlers/assessment-authorization_test.js

```diff
@@ -0,0 +1,56 @@
+import { expect, sinon, hFake } from '../../../test-helper.js';
+import { assessmentAuthorization } from '../../../../lib/application/preHandlers/assessment-authorization.js';
+
+describe('Unit | Pre-handler | Assessment Authorization', function () {
+  let assessmentRepository;
+  let requestResponseUtils;
+  let dependencies;
+
+  beforeEach(function () {
+    assessmentRepository = {
+      getByAssessmentIdAndUserId: sinon.stub(),
+    };
+    requestResponseUtils = { extractUserIdFromRequest: sinon.stub() };
+    dependencies = { assessmentRepository, requestResponseUtils };
+  });
+
+  describe('#verify', function () {
+    describe('When user is the owner of the assessment', function () {
+      it('should return true', async function () {
+        // given
+        const request = {
+          params: {
+            id: 100,
+          },
+        };
+        requestResponseUtils.extractUserIdFromRequest.returns(10);
+        assessmentRepository.getByAssessmentIdAndUserId.withArgs(100, 10).resolves(true);
+
+        // when
+        const response = await assessmentAuthorization.verify(request, hFake, dependencies);
+
+        // then
+        expect(response).to.be.true;
+      });
+    });
+
+    describe('When user is not the owner of the assessment', function () {
```

Comment:
J'ai l'impression que owner n'est pas le bon terme :) 
```suggestion
    describe('When user is not linked to the assessment', function () {
```

Reply:
J'ai regardé et dans le repository c'est bien les termes utilisés "assessmentRepository.ownedByUser"